src/tlog-test-fd-json-reader.c

```diff
@@ -236,7 +236,7 @@ main(void)
          "",
          OP_LOC_GET(1),
          OP_READ(TLOG_RC_OK, NULL),
-         OP_LOC_GET(1),
+         OP_LOC_GET(1),               
```

Comment:
Haha the reason why I have this space because I comment `//1` to recovery and create the fail test by putting another value here. I do not realize it has effect on github. Next time, I will open the correct test case on web page